config/settings.yml

```diff
@@ -1023,6 +1023,7 @@
   :interval: 60
   :memcache_server: 127.0.0.1:11211
   :memcache_server_opts: "-l 127.0.0.1"
```

Comment:
I just checked the memcache code. It uses bytes by default if no "unit" is specified. 

```c
            buf = strdup(optarg);
            unit = buf[strlen(buf)-1];
            if (unit == 'k' || unit == 'm' ||
                unit == 'K' || unit == 'M') {
                buf[strlen(buf)-1] = '\0';
                size_max = atoi(buf);
                if (unit == 'k' || unit == 'K')
                    size_max *= 1024;
                if (unit == 'm' || unit == 'M')
                    size_max *= 1024 * 1024;
                settings.item_size_max = size_max;
            } else {
                settings.item_size_max = atoi(buf);
            }
            free(buf);
```

so an implementation could be:

```patch
diff --git a/app/models/miq_server/environment_management.rb b/app/models/miq_server/environment_management.rb
index 9763a9d..2481bf7 100644
--- a/app/models/miq_server/environment_management.rb
+++ b/app/models/miq_server/environment_management.rb
@@ -78,6 +78,8 @@ module MiqServer::EnvironmentManagement
       require "#{Rails.root}/lib/miq_memcached" unless Object.const_defined?(:MiqMemcached)
       svr, port = cfg.config.fetch_path(:session, :memcache_server).to_s.split(":")
       opts = cfg.config.fetch_path(:session, :memcache_server_opts).to_s
+      value_max_bytes = cfg.config.fetch_path(:session, :memcache_value_max_bytes).to_s
+      opts << " -I #{value_max_bytes}"
       MiqMemcached::Control.restart!(:port => port, :options => opts)
       _log.info("Status: #{MiqMemcached::Control.status[1]}")
     end
```